Update README.md (#50)
FirebaseExtended/make-it-so-android: README.md

* [Enable Anonymous Authentication](https://firebase.google.com/docs/auth/android/anonymous-auth#before-you-begin) in your Firebase project
* [Enable Email/Password Authentication](https://firebase.google.com/docs/auth/android/password-auth#before_you_begin) in your Firebase project
* Run the app using Android Studio Flamingo+ on a device/emulator with API level 21 or above

> * Create your first to-do item, then open the Firestore dashboard on the Firebase console
> * Navigate to the Indexes tab and create a new index with the following configuration:

Reviewer: I'd avoid context switching in the same step (ie. first do X in the app then do Y in the console)
What if we only mention the app in the first step and then mention the console in a second step?

Also, we can deep link to the console for the indexes tab.

I added both suggestions:

```suggestion
* Create your first to-do item in the app
* In the Firebase console, navigate to the [Firestore Indexes tab](https://console.firebase.google.com/project/_/firestore/indexes) and create a new index with the following configuration:
```

Author: This looks great, thanks! I'll commit your suggestions.

  
<img width="691" alt="index-config" src="https://github.com/FirebaseExtended/make-it-so-android/assets/14080129/256ff875-f0d8-41d5-b1ab-8fe2e2a18a4a">

> 

Reviewer: I don't think it's a good idea to add screenshots from the console for specific instructions. It's bad for accessibility, plus it won't get translated, and it might quickly become outdated (eg. the screenshot on the [indexes doc page](https://firebase.google.com/docs/firestore/query-data/indexing#use_the_firebase_console) is already outdated).

Maybe we could have another step that says something like "Create a new composite index for the collection `tasks` with 2 fields: `userId` and `createdAt` (both Ascending)".

Also, I don't think we're using a [`collectionGroup`](https://firebase.google.com/docs/firestore/query-data/queries?#collection-group-query) in the app, right? Should the index only be created for a single "Collection" ? (that last option at the bottom that you have on your screenshot)

Author: Done!

> This is necessary when fetching documents because this app uses `where` and `orderBy` operators in different fields: `where` uses the `userId` field and `orderBy` uses the `createdAt` field (see `StorageServiceImpl`).

Reviewer: Maybe we could link to the docs for folks who are not familiar with indexes.

```suggestion
This is necessary when fetching documents because this app uses `where` and `orderBy` operators in different fields: `where` uses the `userId` field and `orderBy` uses the `createdAt` field (see `StorageServiceImpl`). Learn more on the Firebase documentation about [Index types](https://firebase.google.com/docs/firestore/query-data/index-overview#composite_indexes).
```

Author: Good catch!
